package.json

```diff
@@ -12,6 +12,8 @@
     "electron-rebuild": "electron-rebuild --parallel --force --types prod,dev,optional --module-dir app",
     "flow": "flow",
     "flow-typed": "rimraf flow-typed/npm && flow-typed install --overwrite || true",
+    "format": "prettier --single-quote --list-different '**/*.js'",
```

Comment:
I think the `"format"` script should be removed and the `"format-fix"` should be merged into the `"lint-fix"` script.

Reply:
This will make it easier for existing users to upgrade without changing their workflow

Reply:
done